src/PKSim.Infrastructure/Serialization/Json/NullabeDoubleJsonConverter.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace PKSim.Infrastructure.Serialization.Json
+{
+   public class NullabeDoubleJsonConverter : JsonConverter
+   {
+      private const int DOUBLE_PRECISION = 10;
```

Comment:
here I am saying that I am rounding each value to 10 digits before serializing it. That should be more than enough to avoid any side effects